banner/styles.py

```diff
@@ -0,0 +1,45 @@
+import inspect
```

Comment:
Rather look at https://github.com/praekelt/django-composer-prk/blob/develop/composer/admin.py#L31-L38 for showing the styles. A curated list is less effort to deal with than this implementation.